Podverse/FiltersTableHeaderView.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  FiltersTableHeaderView.swift
+//  Podverse
+//
+//  Created by Creon Creonopoulos on 10/3/17.
+//  Copyright © 2017 Podverse LLC. All rights reserved.
+//
+
+import UIKit
+
+protocol FilterSelectionProtocol {
```

Comment:
Do you need all these options? Is there a way we can have only one sortBy method and pass it the sortingTypeSeleted as an argument? on each of the action sheet options?

Reply:
Hmmm that would make sense. I wasn't sure if we could pass a parameter to a protocol method, but I'll give it a try.